dev-client/src/screens/HomeScreen/components/TemporarySiteCallout.tsx

```diff
@@ -41,6 +41,15 @@ type Props = {
 export const TemporarySiteCallout = ({coords, closeCallout}: Props) => {
   const {t} = useTranslation();
   const navigation = useNavigation();
+  const [siteElevation, setSiteElevation] = useState('');
+
+  useMemo(async () => {
+    const elevation = await getElevation(
+      parseFloat(coords.latitude.toFixed(5)),
+      parseFloat(coords.longitude.toFixed(5)),
+    );
+    setSiteElevation(t('site.elevation_value', {value: elevation, units: 'm'}));
+  }, [coords, t]);
```

Comment:
suggestions:
  - could move the lat/lon rounding into the `getElevation` call
  - it's more idiomatic in our codebase to store the model value (i.e. the actual numeric elevation returned) in state, and convert it to a string while rendering. but if you want to keep it stored as a string, i agree with ruxandra that i expect it to be a number, so maybe you could change the variable name to `elevationString` or something similar?